### PR TITLE
Add heading to index page

### PR DESCRIPTION
**Pull request by Ellon-M**
## Changes

- Initial workflows set-up with Github Actions for HTML & CSS linters.
- Added a heading to the index page with the title "Hello Microverse".
- An external stylesheet `style.css` to style HTML content. 
- Added `test.md` file to be ignored by git.
- Included a `readme.md` file to explain the purpose of this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+# Hello Microverse
+
+> An exercise for mastering all the tools and best practices when setting up a HTML & CSS environment. It involves the creation of feature branches, setting up Github Actions for automating workflows and general knowledge of Git as a version control tool.
+
```
Comment (jerryowusu) [inline README.md:5]: - [ ] You currently do not have a valid and descriptive readme file. :bookmark_tabs:  Kindly consider using [the proffessional](https://github.com/microverseinc/readme-template/blob/master/README.md) template provided by microverse and edit all sections according to your credentials and project. :+1:  Make sure all header/sections are availble.

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Ellon-M :wave: 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

almost-paul-walker


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages
:heavy_check_mark: Excellent descriptive PR title and summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ellon-M) [reply README.md:5]: Thanks! 
All the sections are now added on the readme file as per the template provided.

**Review by levy002 — APPROVED** [review]
### Hello @Ellon-M 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**